db/schema.rb

```diff
@@ -821,6 +821,19 @@
     t.index ["access_grant_id"], name: "index_oauth_openid_requests_on_access_grant_id"
   end
 
+  create_table "paypal_records", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.string "record_id"
+    t.string "status"
+    t.string "payload"
+    t.integer "amount_in_cents"
+    t.string "currency_code"
+    t.string "record_type"
+    t.bigint "parent_record_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["parent_record_id"], name: "index_paypal_records_on_parent_record_id"
+  end
+
```

Comment:
Seems like this slipped in from some other changes?
Note that Rails `db:migrate` consists of a step that dumps the DB schema as is currently present in your local development setup. Rails has no way of telling whether a certain table was created as part of the feature branch/PR you're currently working on, or as one of the other three feature branches which also exist on your machine. That's why you need to be careful with `schema.rb`.